@navikt/ds-react/package.json

```diff
@@ -27,14 +27,13 @@
   },
   "dependencies": {
     "@popperjs/core": "^2.5.4",
-    "@types/react-modal": "^3.10.6",
-    "@navikt/ds-icons": "0.1.0-beta.1",
     "@navikt/ds-css": "0.1.0-beta.1",
+    "@navikt/ds-icons": "0.1.0-beta.1",
     "classnames": "^2.2.6",
-    "react-popper": "^2.2.4",
     "react-collapse": "^5.1.0",
+    "react-merge-refs": "^1.1.0",
     "react-modal": "3.11.2",
-    "react-merge-refs": "^1.1.0"
+    "react-popper": "^2.2.4"
```

Comment:
Får vi til å holde avhengighetene innenfor hver komponent? Kan det løses med yarn workspaces?

Reply:
VI kan løse det slikt for vårt egent oppsett, men går ikke når vi publiserer pakkene da npm/yarn bare vil se på denne root package.json hvis jeg ikke tar helt feil. Er noe vi kan eksperimentere med etterhvert.